tests/test_config.py

```diff
@@ -89,9 +89,14 @@ def test_configfile_hierarchy(tmpdir):
 
     min_c_file = os.path.join(tmpdir, "custom.cfg")
 
+    os.environ["INMANTA_SERVER_AUTH"] = "true"
```

Comment:
These environment variables are not cleaned up after the run of this test case. This might result in undesired side-effects in other test cases. The `Monkeypatching` pytest plugin is a nice tool to solve this issue.

Reply:
we should create a fixture to set them up and clean them up. would monkeypatch provide any additional safeties over unsetenv in the case of a fixture?

Reply:
Monkeypatch doesn't necessarily do an unset of the environment variable. It makes sure that all modification are undone. _"All modifications will be undone after the requesting test function or fixture has finished"_ (see: https://docs.pytest.org/en/latest/monkeypatch.html)

Reply:
resolved